### ISSUE TITLE
[FEATURE] CI/CD pipeline automation?

### ISSUE THREAD

**Issue by meokey** [FlowiseAI/Flowise#1987]
**Describe the feature you'd like**
Is there a plan to automate the docker image deployment? There are so many new features/changes on the codes and new AI models everyday, and we would have to wait until new image created manually to include them 🔢 
Personally I cannt wait to try Claude Haiku on Flowise but it's not included in the 1.6.1 version released last week.

**Additional context**

**Comment by HenryHengZJ** [comment]
Yes! Exactly, we are working on a alternative that let user to specify their own models in a JSON file
